### ISSUE TITLE
Play simple text in iOS Safari

### ISSUE THREAD

**Issue by rjcortese** [Hagsten/Talkify#55]
Hello!
This is just a question about how the lib works on iOS Safari...

I am developing a web based car navigation / routing app and have added turn by turn instructions.
It currently uses the regular Web Speech API that Talkify seems to use underneath (at least for HTML5 player). However, this implementation does not work when the app runs on iOS Safari (afaik because of autoplay issues).

Do you know if Talkify works on iOS Safari, does it have the same limitations as the regular speechSynthesis?
Might it be able to autoplay?

Thanks!

**Comment by Hagsten** [comment]
Hi,

We have not implemented any means to bypass the autoplay limitation. I think autoplay limitation affects all sorts of media and not only the SpeechSynthesis. I have myself added Talkify (with our remote server) on an old iPad running Safari in my Home automation and have bypassed autoplay using a welcome button that the user needs to click on in order to proceed. On that version of Safari that is enough user interaction for autoplay to function.

TLDR;
Talkify should work as good as SpeechSynthesis in regards to autoplay but you will probably need to bypass it by adding a button or something ad page load :)

/Andreas

**Comment by rjcortese** [comment]
Thank you for the quick response! I'll try your suggestion!
